user-registration-app/src/app/app.component.ts

```diff
@@ -59,12 +62,12 @@ export class AppComponent implements AngularFireAuthModule, AfterViewInit {
       .subscribe((event) => {
         switch (event.constructor.name) {
           case 'NavigationStart':
-            this.progressBar.start();
+            this.progressBar.ref().start();
```

Comment:
The newest progress bar package update has them all referencing some global instance